x-pack/plugins/apm/server/lib/errors/get_error_group.ts

```diff
@@ -35,16 +37,9 @@ export async function getErrorGroup({
   const { start, end, esFilterQuery, client, config } = setup;
   const filter: ESFilter[] = [
     { term: { [SERVICE_NAME]: serviceName } },
+    { term: { [PROCESSOR_EVENT]: 'error' } },
```

Comment:
Every query should explicitly specify the document types being targeted.
This both makes the query more resilient to schema changes, and potentially faster